src/Symfony/Component/Security/Http/RateLimiter/DefaultLoginRateLimiter.php

```diff
@@ -31,7 +31,7 @@ final class DefaultLoginRateLimiter extends AbstractRequestRateLimiter
     private $globalLimiter;
     private $localLimiter;
```

Comment:
I suggest to rename these properties/constructor args as well

Reply:
Thank you